### ISSUE TITLE
Handle WMS Layer Scale Dependencies

### ISSUE THREAD

**Issue by mweech** [fgpv-vpgf/fgpv-vpgf#426]
Handle situations where a WMS layer is only available starting at a specific zoom scale (e.g. http://maps.geogratis.gc.ca/wms/hydrography_en?service=wms&version=1.3.0&request=GetCapabilities&layers=hydro_features).  This information is included in the GetCapabilities.

Acceptance criteria:
- WMS 1.3 layers are supported, previous versions will not be in scope
- When a layer is expected to be out of scale, add an indicator to the layer legend to indicate the "WMS layer reports as out of scale and should not be visible at this time."
- RAMP will not take any action to prevent the layer from being viewed if it responds with imagery outside of its defined zoom scale. It will be possible to toggle the show/hide controls regardless of the zoomscale state.

**Comment by mweech** [comment]
From DanB's comments: https://github.com/RAMP-PCAR/RAMP-PCAR/issues/10#issuecomment-110384964

Some technical issues and limitations with regards to this issue:

- WMS 1.1.1 and 1.3.0 have different declarations and interpretations for the zoom scale dependency
- WMS 1.3.0 appears to be close to the same declaration as an ESRI feature service
- Both specs leave the actual result ambiguous as the service is free to return imagery outside of its zoom scale boundaries
- Unlike feature services, RAMP cannot detect that imagery is unavailable (service can return a blank image with no error message)
- WMS recommendations for FGP: all WMSes requiring zoom scale be at v1.3.0 and behave in a consistent manner (actual decision can be made by some committee, but essentially services should either always return imagery outside of the zoom scale area, or never return imagery outside the zoom scale area)
- 

In a nutshell, we can implement support for 1.3.0 WMS services zoom scale widget, but there is nothing preventing the service from serving or not serving data outside the boundaries of this defined range. So this could seem like an issue or a bug when the issue is on the server end of things. We would be happy to receive any feedback or experience with these issues and will avoid addressing this zoom scale issue for WMS for now.

**Comment by mweech** [comment]
When "out of scale" indicator is applied to WMS layers, provide additional feedback that layer may still be visible.  i.e. "Layer asserts it is out of scale.  May still be visible."
